ProcessLib/ProcessVariable.cpp

```diff
@@ -113,4 +114,24 @@ MeshLib::Mesh const& ProcessVariable::getMesh() const
 	return _mesh;
 }
 
+MeshLib::PropertyVector<double>& ProcessVariable::getOrCreateMeshProperty()
+{
+	boost::optional<MeshLib::PropertyVector<double>&> result;
```

Comment:
Both `getPropertyVector()` and `createNewPropertyVector()` return `boost::optional`. So it is convinient, although because it is checked if the property exists one could do without...
This way is a little safer, because you get a chance to check if everything went well before dereferencing.